app/views/cloud_tenant/edit.html.haml

```diff
@@ -9,10 +13,11 @@
       .col-md-8
         %input.form-control{:type          => "text",
                             :name          => "name",
-                            'ng-model'     => "cloudTenantModel.name",
+                            'ng-model'     => "vm.cloudTenantModel.name",
                             'ng-maxlength' => 128,
                             :miqrequired   => false,
-                            :checkchange   => true}
+                            :checkchange   => true
```

Comment:
The only way for haml to know that the line is continuing on the next one is the comma :). So you either need to add a comma or keep the closing bracket on this line.